theme/themes/eea/globals/site.variables

```diff
@@ -124,7 +124,7 @@
 --------------------*/
 
 /* Global focus-visible for accessibility */
-@focusVisibleOutline: .2rem auto var(--focus-visible, @blue-3);
+@focusVisibleOutline: .2rem auto var(--focus-visible, @blue-2);
```

Comment:
@g-stamatis why did you change the focus outline to blue-2 from 3? we reference blue 3 for focus in several files when we style the outline

Reply:
@ichim-david it was a UX decision to avoid having the same colour between WISE F and the focus outline.

Comment has been added to the ticket now.

Reply:
@g-stamatis then we also need to make changes in other packages where we've used @blue-3 for outline such as volto-searchlib so yes it's a must for us to get any changes in outline change as that will need to be then tested and changed everywhere as well

Reply:
@ichim-david please add your comments at the ticket, where the UX team can see them.